extension-dev.code-workspace

```diff
@@ -90,7 +102,7 @@
         "options": {
           "cwd": "${workspaceFolder:Client}"
         },
-        "command": "Invoke-Build Build",
+        "command": "./build.ps1",
```

Comment:
What's the reason for this change?

Reply:
build.ps1 has more scaffolding currently than Invoke-Build does. If we plan to make the invoke-build task all-encompassing and just make build.ps1 an entrypoint then this can change back, I plan to evaluate that stuff in a future PR.

Reply:
Sounds good!